platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/network/topology/StaticTopology.java

```diff
@@ -121,4 +102,19 @@ public boolean shouldConnectTo(final NodeId nodeId) {
     public RandomGraph getConnectionGraph() {
         return connectionGraph;
     }
+
+    /**
+     * Maps the list of peers to a map of node IDs to their index in the peer list
+     * and populates the peerNodesList with the node IDs
+     *
+     * @param peers the list of peers
+     * @return the map of node IDs to their peer Id
+     */
+    @NonNull
+    private Map<NodeId, Long> map(@NonNull final List<PeerInfo> peers) {
+        for (final PeerInfo peer : peers) {
+            peerNodeToIdMap.put(peer.nodeId(), peer.nodeId().id());
```

Comment:
This definition of a map is not meaningful.  If you have the key, then you can call `.id()` and get the value without using the map.  

I think your intent may be to create a new index ordering based on PeerInfo and map to indexes for use in compatibility with RandomGraph?

Reply:
outdated. Resolving